src/OpenTelemetry.Api/README.md

```diff
@@ -392,8 +392,26 @@ corresponding overloads of `ActivityEvent`.
 
 OpenTelemetry defines a concept called
 [Status](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#set-status)
-to be associated with `Activity`. There is no `Status` class in .NET, and hence
-`Status` is set to an `Activity` using the following special tags:
+to be associated with `Activity`. Starting with [DiagnosticSource
+6.0](https://www.nuget.org/packages/System.Diagnostics.DiagnosticSource/6.0.0),
+`SetStatus` API on `Activity` may be used to set status and status description
+as shown below:
+
+```csharp
+activity?.SetStatus(ActivityStatusCode.Ok);
+activity?.SetStatus(ActivityStatusCode.Error, "Error Description");
+```
+
+All the official
+[Exporters](../../docs/trace/extending-the-sdk/README.md#exporter) shipped from
+this repo supports the above shown mechanism of setting status.
+
+#### Setting Status - DiagnosticSource version older than 6.0
+
+Prior to [DiagnosticSource
```

Comment:
Do we even need this? (e.g. the 1.2.0 version of the SDK would require S.D.DS >= 6.0?)

Reply:
leaving it here until 1.2 stable is out.